lib/interfaces/express.js

```diff
@@ -25,66 +25,56 @@ var expressRequestInformationExtractor =
 var errorHandlerRouter = require('../error-router.js');
 
 /**
- * The Express Error Handler function is an interface for the error handler
- * stack into the Express architecture. This function accepts four arguments:
- * a bound reference to a client which should be given by the proceeding
- * handlerSetup function, an error argument which can be of any type, an
- * express request object, an express response object and the express next
- * function for passing input up the plugin chain.
- * @function expressErrorHandler
- * @param {AuthClient} client - a bound and inited Auth Client instance
- * @param {NormalizedConfigurationVariables} config - the environmental
+ * Returns a function that can be used as an express error handling middleware.
+ * @function makeExpressHandler
```

Comment:
Google style guide doesn't require [at]function, but it allows for it: https://google.github.io/styleguide/javascriptguide.xml?showone=Comments#Comments. 

With that style guide given, I think we should favour DRY. The guideline Matt pointed to makes sense to me.